yarn.lock

```diff
@@ -5858,15 +5858,16 @@ hide-powered-by@1.0.0:
   integrity sha1-SoWtZYgfYoV/xwr3F0oRhNzM4ys=
 
 history@^4.7.2:
```

Comment:
I think we should increase the minimal required version in the package.json to `^4.9.0` as well.